src/main/scala-2.12/com/linagora/gatling/imap/protocol/ImapSessions.scala

```diff
@@ -139,6 +139,8 @@ private class ImapSession(client: IMAPClient, protocol: ImapProtocol) extends Ba
     case msg@Command.Disconnect(userId) =>
       session.disconnect()
       context.become(disconnecting(sender()))
+    case message =>
+      logger.error(s"connected - got unexpected message $message")
```

Comment:
well, i'm not comfortable with that solution: where does this command come from? can it be a server problem? If so, should we report it in the gatling report?

Reply:
It's when we receive a ```Command.connect``` message while we are already connected.
We encounter this case before our ImapAuthenticationIT in the james-gatling project is faulty. It try to connect in a duration loop so it works the first time and fail afterward.

I prefer to log it as it's done in all the other context of this actor than crashing the actor.

I have just added a new to commit to allow to disconnect from the imap session in the gatling scenarii

Reply:
Ok, I didn't remind that we already handle error that way.
It's probably not a good choice but it makes sense to keep it that way for consistency.